client/components/gsuite/docs/new-user-list.tsx

```diff
@@ -92,4 +92,6 @@ const GSuiteNewUserListExample = () => {
 	);
 };
 
+GSuiteNewUserListExample.displayName = 'GSuiteNewUserListExample';
```

Comment:
This should go on the wrapper component (`./gsuite/docs/example`) instead. :)

<img width="534" alt="Screen Shot 2020-01-22 at 9 11 22 PM" src="https://user-images.githubusercontent.com/8892849/72951522-a6074000-3d5c-11ea-865c-0a7366378951.png">

Reply:
Indeed, thanks! Pushed a fix.